src/containers/profile/src/edit/EditAddressForm.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+import React, { useEffect, useMemo, useState } from 'react';
+
+import { List, Map } from 'immutable';
+import { DataProcessingUtils, Form } from 'lattice-fabricate';
+import { DataUtils, LangUtils, ReduxUtils } from 'lattice-utils';
+import type { UUID } from 'lattice';
+
+import { AppTypes, PropertyTypes } from '../../../../core/edm/constants';
+import {
+  APP,
+  APP_REDUX_CONSTANTS,
+  EDM,
+  PROPERTY_TYPE_IDS,
+  ProfileReduxConstants,
+  REQUEST_STATE,
+} from '../../../../core/redux/constants';
+import { selectPerson } from '../../../../core/redux/selectors';
+import { useDispatch, useSelector } from '../../../app/AppProvider';
+import { SUBMIT_ADDRESS, editAddress, submitAddress } from '../actions';
+import { schema, uiSchema } from '../schemas/EditAddressSchemas';
+
+const { LOCATION, LOCATED_AT, PEOPLE } = AppTypes;
+const {
+  LOCATION_ADDRESS,
+  LOCATION_ADDRESS_LINE_2,
+  LOCATION_CITY,
+  LOCATION_STATE,
+  LOCATION_ZIP,
+} = PropertyTypes;
+const { PERSON_NEIGHBOR_MAP, PROFILE } = ProfileReduxConstants;
+const { getEntityKeyId, getPropertyValue } = DataUtils;
+const {
+  getEntityAddressKey,
+  getPageSectionKey,
+  processAssociationEntityData,
+  processEntityData,
+} = DataProcessingUtils;
+const { isDefined } = LangUtils;
+const { isPending } = ReduxUtils;
+
+const EditAddressForm = () => {
+  const dispatch = useDispatch();
+
+  const addressList :List = useSelector((store) => store.getIn([PROFILE, PERSON_NEIGHBOR_MAP, LOCATION], List()));
+  const address :Map = addressList.get(0, Map());
+
+  const populatedFormData = useMemo(() => ({
+    [getPageSectionKey(1, 1)]: {
+      [getEntityAddressKey(0, LOCATION, LOCATION_ADDRESS)]: getPropertyValue(address, [LOCATION_ADDRESS, 0]),
+      [getEntityAddressKey(0, LOCATION, LOCATION_ADDRESS_LINE_2)]: getPropertyValue(
+        address, [LOCATION_ADDRESS_LINE_2, 0]
+      ),
+      [getEntityAddressKey(0, LOCATION, LOCATION_CITY)]: getPropertyValue(address, [LOCATION_CITY, 0]),
+      [getEntityAddressKey(0, LOCATION, LOCATION_STATE)]: getPropertyValue(address, [LOCATION_STATE, 0]),
+      [getEntityAddressKey(0, LOCATION, LOCATION_ZIP)]: getPropertyValue(address, [LOCATION_ZIP, 0]),
+    }
```

Comment:
suggested change here to make this more readable / take advantage of similarity in structure: 

example: 
```
const fqns = [LOCATION, LOCATION_ADDRESS, LOCATION_ADDRESS_LINE_2, LOCATION_CITY, LOCATION_STATE, LOCATION_ZIP]

const eaks = fqns.reduce((obj, fqn) => ({
    [getEntityAddressKey(0, LOCATION, fqn)]: getPropertyValue(address, [fqn, 0]),
     ...obj
}), {})

return {
  [getPageSectionKey(1, 1)]: eaks
} 
```

or use immutable's setIn

```
const result = {}
const psk = getPageSectionKey(1, 1)

fqns.forEach((fqn) => {
    const eak = getEntityAddressKey(0, LOCATION, fqn)
    result = setIn(result, [psk, eak], getPropertyValue(address, [fqn, 0])
})

return result
```